ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/BindingVerifier.kt

```diff
@@ -45,9 +43,10 @@ abstract class BindingVerifier(val httpResponse: Response) {
         }
 
         /** Verifies the response's and assertions' signatures */
-        fun verifyXmlSignature(samlResponse: String) {
+        fun verifyXmlSignature(dom: Document) {
```

Comment:
✏️ Since this method verifies more than 1 signature, it might make more sense to name it plural: `verifyXmlSignatures`.